packages/flutter/test/rendering/debug_test.dart

```diff
@@ -30,14 +31,8 @@ void main() {
     expect(simple.name, equals('transform'));
     expect(simple.value, same(transform));
     expect(
-      simple.toString(),
-      equals(
-        'transform:\n'
-            '  [0] 2.0,0.0,0.0,0.0\n'
-            '  [1] 0.0,2.0,0.0,0.0\n'
-            '  [2] 0.0,0.0,2.0,0.0\n'
-            '  [3] 0.0,0.0,0.0,1.0',
-      ),
+      simple.toString(parentConfiguration: singleLineTextConfiguration),
+      equals('transform: [2.0,0.0,0.0,0.0; 0.0,2.0,0.0,0.0; 0.0,0.0,2.0,0.0; 0.0,0.0,0.0,1.0]'),
```

Comment:
can we verify that in anything that isn't explicitly single-line, this still looks like a matrix?